azure-pipelines.yml

```diff
@@ -863,3 +864,15 @@ jobs:
 #     fetchDepth: 25
 #   - bash: ./checker/bin-devel/test-downstream.sh
 #     displayName: test-downstream.sh
+
+- job: windows_junit_jdk17
+  dependsOn:
+    - junit_jdk21
+  pool:
+    vmImage: 'windows-latest'
+  timeoutInMinutes: 90
+  steps:
+    - checkout: self
+      fetchDepth: 25
+    - bash: ./checker/bin-devel/test-cftests-junit.sh
```

Comment:
@WeilanTao I don't know how I broke the build, but CI doesn't run anymore. Do you see what's wrong?

Reply:
I now manually triggered a build of the latest commit... no idea why it didn't start automatically.